README.md

```diff
@@ -293,10 +287,25 @@ installed.
 1. [Build](#build) the changes and push them to some tag
 2. Use the new tag in your workflows:
 
-```diff
--uses: paritytech/pr-custom-review@1
-+uses: paritytech/pr-custom-review@2
-```
+    ```diff
+    -uses: paritytech/pr-custom-review@1
+    +uses: paritytech/pr-custom-review@2
+    ```
+
+### Deployment
+
+1. Create a token with `workflow`, `read:org`, `user` scopes.
+
+    ![Token scopes](./img/token-scopes.png)
+
+    and save it into the repo or orga secrets.
+
+    Use this token in `with: token:` of the Action definition.
```

Comment:
This is demonstrated in https://github.com/paritytech/pr-custom-review/blob/e52ae0a982c6b3c3f89336b2ceb5ca2e2b34bb35/README.md?plain=1#L208 already